AnnoyancesFilter/sections/popups.txt

```diff
@@ -90,6 +90,8 @@ sodapdf.com##.be-tooltip
 !
 ibtimes.com##.grwf2_backdrop
 ibtimes.com##.grwf2-wrapper
+znanija.com#?##__next > div.js-main-page > div[class^="section"]:has(> div[class^="section__content"] > div.sg-flex > div.sg-flex > button.sg-button > span.sg-button__text:contains(Подпишись))
```

Comment:
Please test this rule instead of ext-css `znanija.com###__next > div.js-main-page > div:not([class]) + div[class^="section--"]`